package.json

```diff
@@ -7,7 +7,6 @@
     "changelog-maker": "./changelog-maker.js"
   },
   "author": "Rod <rod@vagg.org> (https://r.va.gg/)",
-  "license": "MIT",
```

Comment:
Why was this removed?

Reply:
yeah, good question! I have no idea, but I'll rectify.